source/tests/universal/pt/model/test_model.py

```diff
@@ -0,0 +1,296 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import unittest
+
+from deepmd.pt.model.atomic_model import (
+    DPAtomicModel,
+    PairTabAtomicModel,
+)
+from deepmd.pt.model.descriptor import (
+    DescrptDPA1,
+    DescrptDPA2,
+    DescrptHybrid,
+    DescrptSeA,
+    DescrptSeR,
+    DescrptSeT,
+)
+from deepmd.pt.model.model import (
+    DipoleModel,
+    DOSModel,
+    DPZBLModel,
+    EnergyModel,
+    PolarModel,
+)
+from deepmd.pt.model.task import (
+    DipoleFittingNet,
+    DOSFittingNet,
+    EnergyFittingNet,
+    PolarFittingNet,
+)
+
+from ....consistent.common import (
+    parameterized,
+)
+from ...common.cases.model.model import (
+    DipoleModelTest,
+    DosModelTest,
+    EnerModelTest,
+    PolarModelTest,
+    ZBLModelTest,
+)
+from ..backend import (
+    PTTestCase,
+)
+from ..descriptor.test_descriptor import (
+    DescriptorParamDPA1,
+    DescriptorParamDPA2,
+    DescriptorParamHybrid,
+    DescriptorParamHybridMixed,
+    DescriptorParamSeA,
+    DescriptorParamSeR,
+    DescriptorParamSeT,
+)
+
+
+@parameterized(
+    (
+        (DescriptorParamSeA, DescrptSeA),
+        (DescriptorParamSeR, DescrptSeR),
+        (DescriptorParamSeT, DescrptSeT),
+        (DescriptorParamDPA1, DescrptDPA1),
+        (DescriptorParamDPA2, DescrptDPA2),
+        (DescriptorParamHybrid, DescrptHybrid),
+        (DescriptorParamHybridMixed, DescrptHybrid),
+    )  # class_param & class
+)
+class TestEnergyModelPT(unittest.TestCase, EnerModelTest, PTTestCase):
+    # @property
+    # def modules_to_test(self):
+    #     # for Model, we can test script module API
+    #     modules = [
+    #         *PTTestCase.modules_to_test.fget(self),
+    #         self.script_module,
```

Comment:
Why is it commented?

Reply:
> May need help:
> script module API test in model uts are excluded because of long processing time. I've tried to only get torch.jit.script once, but it still doesn't work.